docs/report_identification.md

```diff
@@ -0,0 +1,28 @@
+# Analyzer report identification
+
+Unique report idetifiers are required to compare analysis results.
+If the static analyzers (Clang Static Analyzer, Clang Tidy) do not generate
+a unique report identifier, CodeChecker tries to generate a unique id.
+
+It is recommended to use the identifier generated by the static analyzers
+because they can include the semantic context better of the result into the
```

Comment:
word order is mishap here.

because they can include the semantic context of the result better into the